model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/entities/LoginFailureEntity.java

```diff
@@ -28,8 +29,12 @@
  * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
  */
 @ProtoTypeId(Marshalling.LOGIN_FAILURE_ENTITY)
+@Indexed
 public class LoginFailureEntity extends SessionEntity {
 
+    // Increment this version if any indexed field is added, removed, or changed.
+    public static final int INDEX_VERSION = 1;
```

Comment:
This seems to be unused? Is this some relict from other approach to reindexing per cache?